<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatRoom</class>
 <widget class="QMainWindow" name="ChatRoom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1436</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* QPushButton - 发送按钮 */
QPushButton, QToolButton {
    background: #E8E8E8;
    border: none;
    color: 	#000000;
    font-size: 14px;
    padding: 4px 8px;
    outline: none;
}

QPushButton:hover, QToolButton:hover {
    background: #f0f0f0;
    border-radius: 3px;
}

QPushButton:pressed, QToolButton:pressed {
    background: #dcdcdc;
}

/* QListView - 消息列表 */
QListView {
    background: #ffffff;
    border: none;
    outline: none;
    font-size: 14px;
    color: #222222;
    padding: 5px;
}

/* QTextEdit - 聊天输入框 */
QTextEdit {
    border: 1px solid #cccccc;
    border-radius: 3px;
    background: #fafafa;
    font-size: 14px;
    color: #333333;
    padding: 4px;
    outline: none;
}

QTextEdit:focus {
    border: 1px solid #999999;
    background: #ffffff;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="TimeLabel">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton">
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/new/prefix1/imgs/x.png</normaloff>:/new/prefix1/imgs/x.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QListView" name="listView"/>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>140</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QToolButton" name="toolButton_file">
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>28</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>16</height>
              </size>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="icons.qrc">
               <normaloff>:/new/prefix1/imgs/a61f9036e72089fb21c888e21ad03967.jpg</normaloff>:/new/prefix1/imgs/a61f9036e72089fb21c888e21ad03967.jpg</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonFollowStyle</enum>
             </property>
             <property name="autoRaise">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButton_video">
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>28</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>16</height>
              </size>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="icons.qrc">
               <normaloff>:/new/prefix1/imgs/856fc6535b20b341d3bc7600799395f3.jpg</normaloff>:/new/prefix1/imgs/856fc6535b20b341d3bc7600799395f3.jpg</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButton_share">
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>28</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>16</height>
              </size>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="icons.qrc">
               <normaloff>:/new/prefix1/imgs/8302bcf5dbd2673ec4fea909024a2ef9.jpg</normaloff>:/new/prefix1/imgs/8302bcf5dbd2673ec4fea909024a2ef9.jpg</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButton_2">
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>28</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>43</width>
               <height>16</height>
              </size>
             </property>
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string>控制设备</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>保存路径选择</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>停止</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>录制</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_emission">
           <property name="text">
            <string>发送</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_chat">
         <property name="minimumSize">
          <size>
           <width>1400</width>
           <height>899</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>1401</width>
           <height>900</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1436</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
